rts/lra/pom.xml

```diff
@@ -142,21 +136,6 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
the checkstyle should not be removed from the project. It could be used the one configured as default for the Narayana project (JBTM-989). The changes which are needed are to to define plugin here in `pom.xml` to be

```xml
 <plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-checkstyle-plugin</artifactId>
  <configuration>
    <skip>false</skip>
  </configuration>
</plugin>
```
and change the version of the wfly checkstyle plugin to version `1.0.8.Final` at the root `pom.xml`
https://github.com/jbosstm/narayana/blob/5.9.5.Final/pom.xml#L491

Reply:
That what I wanted to happen so good spot, thanks